Game/src/Map.cpp

```diff
@@ -70,27 +71,27 @@ void Map::InitBlocks()
     {
       switch (map_[y * width() + x]) {
```

Comment:
I think that is a reason to use "Factory" and "Flyweight"  patterns.

I do by this way in the featured version